### ISSUE TITLE
Reading: Time Masking for Temporal Language Models

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#219]
## 0. Paper
authors: Guy D. Rosin, Ido Guy, Kira Radinsky
paper: [arxiv](https://arxiv.org/abs/2110.06366)

## 1. What is it?
They propose a post-pretraining method of Language Models for temporal analysis.

### 3.1 Method
<img width="1143" alt="スクリーンショット 2021-11-17 2 32 52" src="https://user-images.githubusercontent.com/45454055/142036075-c14ea3ad-ac50-4a6b-83db-b8ad99ed23d3.png">
They propose a post-pretrain method of pre-trained BERT model.

- Training: add a special time token (e.g. <2021>) to each sentence.
- Inference: 
   - time prediction: predict masked time token
   - time-dependent MLM: predict the masked token

### 3.2 Metrics (Lexical Semantic Change)
- temporal-cosine: cosine similarity
<img width="546" alt="スクリーンショット 2021-11-17 2 38 31" src="https://user-images.githubusercontent.com/45454055/142036961-037bb8c0-7e9c-49fa-bb64-8388778e72b8.png">

- temporal-diff:the absolute difference of the probabilities
<img width="546" alt="スクリーンショット 2021-11-17 2 38 45" src="https://user-images.githubusercontent.com/45454055/142036996-1801ec64-3b82-4f40-aad1-f1f5635779b1.png">

### 4.1 Lexical Semantic Change Detection
From Table 3, their model TempoBERT achieved stronger and comparable results to strong baselines.
<img width="1137" alt="スクリーンショット 2021-11-17 2 39 58" src="https://user-images.githubusercontent.com/45454055/142037202-4ef0cebf-640a-4bab-95ec-b2f620dc1eb7.png">

### 4.2 Sentence Time Prediction
From Table 6, their model achieved state-of-the-art performance.
<img width="548" alt="スクリーンショット 2021-11-17 2 45 09" src="https://user-images.githubusercontent.com/45454055/142037986-3d332d5b-ffce-4df9-a169-0bd1ec19216a.png">

### 5.1 Long-term and Short-term semantic shift
Table 4 shows
- time-diff achieved high performance in Short-term shift (LiverpoolFC): based on sentence-time prediction
- time-cos achieved high performance in Long-term shift (SemEval): in long-term, writing style is changed → cosine is less affected by writing style
<img width="548" alt="スクリーンショット 2021-11-17 2 40 14" src="https://user-images.githubusercontent.com/45454055/142037228-a7748fef-3891-46bb-b698-89a2eab1917f.png">

### 5.2 Size of BERT models
From Table 5, TempoBERT-tiny also achieves comparable performance.
This result means understanding time there is no need to use extremely large models.
<img width="548" alt="スクリーンショット 2021-11-17 2 45 50" src="https://user-images.githubusercontent.com/45454055/142038081-d3540c9f-2956-4b97-8155-913f8ce0c22b.png">
